src/tests/ftest/util/ddb_utils_base.py

```diff
@@ -4,23 +4,26 @@
   SPDX-License-Identifier: BSD-2-Clause-Patent
 """
 from command_utils_base import PositionalParameter
-from command_utils import ExecutableCommand
+from command_utils import TargetedExecutableCommand
 
 
-class DdbCommandBase(ExecutableCommand):
+class DdbCommandBase(TargetedExecutableCommand):
     """Defines the basic structures of ddb command."""
 
-    def __init__(self, path):
+    def __init__(self, server_host, path):
```

Comment:
```suggestion
    def __init__(self, server_host, path, verbose=True, timeout=60, sudo=True):
```

And these would all need to be assigned in this function, e.g. `self.sufo=sudo`

Reply:
Done